Server/Temp.Domain/EventMessaging/Contracts/IDomainEvent.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ModularHouse.Server.Temp.Domain.EventMessaging.Contracts;
+
+public interface IDomainEvent
+{
+    Guid TransactionId { get; }
+}
```

Comment:
We can push this property globaly in the context of our logger.
This feature helps us avoid pushing transactionId every time we create new event.

Reply:
Should be implemented in Issue #12